packages/consumption/test/modules/requests/OutgoingRequestsController.test.ts

```diff
@@ -258,8 +270,6 @@ describe("OutgoingRequestsController", function () {
             });
 
             test("create an outgoing request from relationship creation with an active relationship", async function () {
```

Comment:
now the title states that this function should be called using an active relationship but in beforeEach you run `await Given.aPendingRelationshipToIdentity();` which sets a pending relationship instead of an active.